src/lib/Core.jsx

```diff
@@ -166,6 +166,15 @@ function Core({
     );
   };
 
+  const isCorrectCheck = (index, correctA) => {
+    if (typeof correctA === 'string') {
+      return index === Number(correctA);
+    } if (typeof correctA === 'object') {
+      return String(correctA).includes(String(index));
```

Comment:
it only works with single digit. A failing case: `correctA = [11], index = 1`.

Reply:
Fixed issue